CodeEdit/Features/SourceControl/Views/SourceControlAddExistingRemoteView.swift

```diff
@@ -71,4 +66,22 @@ struct SourceControlAddRemoteView: View {
             }
         }
     }
+
+    func submit() {
+        Task {
+            do {
+                try await sourceControlManager.addRemote(name: name, location: location)
```

Comment:
Might be a good idea to add a quick `.trimmingWhitespace` on the name and location here. I found that both text fields allowed for pasting newlines and didn't easily show that there was one at the end. 

Actually this is maybe an input validation problem rather than adding it here. I'll leave that decision to you.